src/fprime/common/models/serialize/bool_type.py

```diff
@@ -44,5 +44,11 @@ def deserialize(self, data, offset):
         except struct.error:
             raise DeserializeException("Not enough bytes to deserialize bool.")
 
-    def getSize(self):
+    @classmethod
+    def getSize(cls):
         return struct.calcsize("B")
```

Comment:
I'm just wondering if this `"B"` should be switched to `"?"` throughout this file. I understand both have a standard size of 1 byte, but `"?"` is for boolean, which is the type we are dealing with in this file.

Reply:
As it turns out we are using a `U8` as our serialize type for booleans. (`0xFF` and `0x00` being the two values).  Thus I'd keep it as-is because the U8 is on-wire type.